back/engines/commercial/report_builder/app/services/report_builder/report_publisher.rb

```diff
@@ -14,7 +14,8 @@ def publish
     craftjs_json = @report.layout.craftjs_jsonmultiloc
     return if craftjs_json.blank?
 
-    nodes = craftjs_json['en']
+    # TODO: change when we use multiple locales
+    nodes = craftjs_json.values.first
```

Comment:
The multiple locale thing has been merged, so this can be changed now I think?

Reply:
Good point. Fixed.